.github/auto-merge.yml

```diff
@@ -0,0 +1,3 @@
+- match:
+    dependency_type: all
+    update_type: 'semver:major'
```

Comment:
Does that mean that major version updates will be merged, too?

Reply:
Ok, changed it to `semver:minor`.